src/components/BackgroundIcon/BackgroundIcon.stories.tsx

```diff
@@ -5,20 +5,68 @@ import {withDesign} from "storybook-addon-designs";
 import {faSearch, faStar} from "@fortawesome/pro-solid-svg-icons";
 import {faStar as faStarLight} from "@fortawesome/pro-light-svg-icons";
 
-// More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
+const summary =
+  "##Summary\nThe Background icon component may contain various icons to visualize or to give information as an image.\n\n";
+const context =
+  "##Context\nThe Background component was added to create a visual of the information it is trying to portray.\n\n";
```

Comment:
Suggestion; I'm not sure if this context is useful, can I suggest we explain the purpose of the component here?  Wemay need some input from UX.